client/src/components/App.jsx

```diff
@@ -13,38 +13,110 @@ const Header = styled.h3`
   text-align: center;
 `;
 
+const AppDisplay = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ButtonLeft = styled.button`
+top: 34%;
+background: white;
+color: black;
+font-size: 1em;
+font-weight: bold;
+z-index: 10;
+padding: 1.25rem 1rem;
```

Comment:
Though probably not impacting you right now, keep in mind it's not great practice to use rem/em/percentages/px all in the same file as it tends to skew things as window sizes change.